catowser/catowser/Search/SearchBarViewV2.swift

```diff
@@ -0,0 +1,49 @@
+//
+//  SearchBarViewV2.swift
+//  catowser
+//
+//  Created by Andrei Ermoshin on 1/3/23.
+//  Copyright © 2023 andreiermoshin. All rights reserved.
+//
+
+import SwiftUI
+
+struct SearchBarViewV2: View {
```

Comment:
This is not used for now, but it will be in next PR for clean SwiftUI without using legacy UIKit views